bugbug/similarity.py

```diff
@@ -349,3 +349,30 @@ def get_similar_bugs(self, query):
             for similar in sorted(similarities, key=lambda v: v[1])[:10]
             if similar[0] != query["id"] and similar[1] < self.cut_off
         ]
+
+    def get_distance(self, query1, query2):
```

Comment:
You are duplicating quite a bit of code between this function and `get_similar_bugs`, I suggest writing an helper function which you can call from both.